pkgs/applications/graphics/screentest/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, automake
+, intltool
+, libtool
+, pkg-config
+, gtk2
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs {
```

Reply:
could you explain this?

Reply:
Yeah sure, Drupal has some good links in his comment here: https://github.com/NixOS/nixpkgs/pull/242774#discussion_r1263382624

Reply:
ah, i hadn't noticed the removal of `rec`, interesting

note that this `rec` is due to `nix-init`, if this is the preferred approach, maybe it should be implemented there

the manual still shows `rec`,
nixpkgs has 49166 occurrences of `mkDerivation` , but "only" 646 of `mkDerivation (finalAttrs:`
and i'm not a fan of having to carry open parenthesis context throughout a full package definition

i'm not going to make this suggested change because it's not an approach i'm familiar with
which is to say, as i'm listed as the maintainer and this is not how i would write the package even after being informed of this option, i prefer to the style i'm used to reading
same with `meta = with lib;`
\</curmudgeon>

Reply:
Sure that's totally fine. I am mostly putting forward what I had been recommended to do elsewhere. I am still learning what is acceptable style differences/opinions here with nix. I apologize if this came across as authoritative in any way.